TESTS/mbed_drivers/timerevent/main.cpp

```diff
@@ -39,13 +39,16 @@ class TestTimerEvent: public TimerEvent {
 public:
     TestTimerEvent() :
             TimerEvent(), sem(0, 1) {
+
+        sleep_manager_lock_deep_sleep_internal();
```

Comment:
You shouldn't use internal versions, but the public macro.

Reply:
Fixed.